lib/compilers/Wine-VC.js

```diff
@@ -36,19 +36,17 @@ class CLCompiler extends BaseCompiler {
 
     exec(command, args, options) {
         options = options || {};
-        if ((process.platform === "linux" || process.platform === "darwin") && command.toLowerCase().endsWith(".exe")) {
-            args.unshift(command);
-            command = this.env.ceProps("wine");
-            options.env = options.env || {};
-            // Force use of wine vcruntime (See change 45106c382)
-            options.env.WINEDLLOVERRIDES = "vcruntime140=b";
-            options.env.WINEDEBUG = "-all";
-        }
+        args.unshift(command);
+        command = this.env.ceProps("wine");
+        options.env = options.env || {};
+        // Force use of wine vcruntime (See change 45106c382)
+        options.env.WINEDLLOVERRIDES = "vcruntime140=b";
```

Comment:
this will cause issues because the vc runtime is not actually installed on the Wine environment, unless you know a way how to install it of course

Reply:
This was here already...

Reply:
i see, strange. oh right, 'b' means builtin. nevermind

Reply:
It was originally in the `compiler-wrapper` script : see 45106c38203301814ed40ee2253110f6709e46e4 by a certain @partouf :)